operators.js

```diff
@@ -31,17 +32,25 @@ function toBufferOrFalsy(value) {
   return Buffer.isBuffer(value) ? value : Buffer.from(value)
 }
 
+const slowEqualsCache = new LRU({
+  max: 1500,
+})
```

Comment:
Max 1500 is a lot, I suspect that we will never pass that threshold, making this LRU effectively the same as a (ES6) `Map`. Can I suggest that you change this `LRU` for a `Map`? That would also help with the bundle size to not include the lru-cache library. Also, the name of this cache could be `seekFromDescCache` because this is used also for `slowIncludes`, not just `slowEquals`.

Otherwise, this PR seems like a nice and simple improvement to perf.

Reply:
@staltz Renamed & switched to `Map`. I favored LRU to support use cases with lots of dynamic keys, but perhaps that can wait until someone needs to address such cases.